### PR TITLE
Findings matcher text improvements

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

### CODE REVIEW

File: model/src/test/kotlin/utils/FindingsMatcherTest.kt [hunk 1]
```diff
@@ -84,12 +81,13 @@ class FindingsMatcherTest : WordSpec() {
 
         "getClosestCopyrightStatements()" should {
             "return exactly the statements within the line threshold" {
-                val licenseStartLine = THRESHOLD + 20
+                // Use an arbitrary license start line that is larger than TOLERANCE_LINES_DEFAULT_VALUE.
+                val licenseStartLine = 20
```
Comment (mnonnenmacher) [inline model/src/test/kotlin/utils/FindingsMatcherTest.kt]: To protect against changes in the default value it makes sense to generate the value here, like before.

File: model/src/test/kotlin/utils/FindingsMatcherTest.kt [hunk 2]
```diff
@@ -54,17 +54,13 @@ private fun createCopyrightFinding(statement: String, path: String, line: Int =
     )
 
 class FindingsMatcherTest : WordSpec() {
-    companion object {
-        private const val LICENSE_FILE_PATH = "a/LICENSE"
-    }
-
-    private val matcher = FindingsMatcher(LicenseFileMatcher(LICENSE_FILE_PATH))
+    private val matcher = FindingsMatcher(LicenseFileMatcher("a/LICENSE"))
```
Comment (mnonnenmacher) [inline model/src/test/kotlin/utils/FindingsMatcherTest.kt]: Commit message nit: "compantion"

Comment (sschuberth) [reply model/src/test/kotlin/utils/FindingsMatcherTest.kt]: Done.

File: model/src/test/kotlin/utils/FindingsMatcherTest.kt (hunk 1 above)
Comment (sschuberth) [reply model/src/test/kotlin/utils/FindingsMatcherTest.kt]: Done in a follow-up commit to avoid conflicts during rebase.

**Pull request merged**